website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -84,15 +84,17 @@ $ curl \
 {
   "data": {
     "cas_required": false,
-    "max_versions": 0,
-    "delete_version_after": "3h25m19s"
+    "delete_version_after": "3h25m19s",
+    "max_versions": 0
   }
 }
 ```
 
 ## Read Secret Version
 
-This endpoint retrieves the secret at the specified location.
+This endpoint retrieves the secret at the specified location. The metadata returned
+here ( `created_time, destroy,` and `version`) is version specific. It should not be
+confused with the response from the [metadata endpoint](/api/secret/kv/kv-v2#read-secret-metadata).
```

Comment:
Is this the correct way to do an anchor? I looked at other files and it appears this is the pattern, but wanted to double check.

Reply:
Yep! looks correct